resources/views/pages/user-shifts.html

```diff
@@ -52,7 +52,7 @@ <h1>%title%</h1>
             </div>
             <div class="row d-print-none">
                 <div class="col-md-12">
-                    <input class="btn btn-primary" type="submit" style="width:100%; margin: 1em 0" value="%filter%">
+                    <button class="btn btn-primary" type="submit" style="width:100%; margin: 1em 0" value="">%filter%</button>
```

Comment:
You can probably remove the value="", too.

Reply:
thx overlooked that. fixed